libs/@local/hash-graphql-shared/src/graphql/type-defs/knowledge/page.typedef.ts

```diff
@@ -68,6 +68,10 @@ export const pageTypedef = gql`
       The account owning the pages. Defaults to the logged in user.
       """
       ownedById: OwnedById
+      """
+      Whether or not to include archived pages. Defaults to false.
+      """
+      includeArchived: Boolean
```

Comment:
We can express the default in the definition like so. 

Unfortunately the coden doesn't understand that there's a default so the types will still have it as undefined, but the actual value passed to the function will be `false` (codegen may have been updated to handle this, haven't checked in a while)
```suggestion
      includeArchived: Boolean = false
```